### PR TITLE
hello from ali

### PR DESCRIPTION
**Pull request by alzaar**
pushed hello

### CODE REVIEW

File: person.rb [hunk 1]
```diff
@@ -2,4 +2,10 @@ class Person
   def initialize(name)
     @name = name
   end
+
+  def hello
+  puts hello
```
Comment (VKorelsky) [inline person.rb]: check your indentation

File: person.rb [hunk 2]
```diff
@@ -2,4 +2,10 @@ class Person
   def initialize(name)
     @name = name
   end
+
+  def hello
```
Comment (Koncoder) [inline person.rb]: Identation dude!!

Comment (aaronjosephm) [reply person.rb]: Indentation*

Comment (VKorelsky) [reply person.rb]: **indentation, dude !

**Pull request merged**